db/badgerdb/badger.go

```diff
@@ -0,0 +1,153 @@
+package badgerdb
+
+import (
+	"os"
+
+	"github.com/dgraph-io/badger/v3"
+	"go.vocdoni.io/dvote/db"
+)
+
+// MEMTABLESIZE defines the BadgerDB maximum size in bytes for memtable table.
+// The default is 64<<20 (64MB), this does not pre-allocate enough memory for
+// big Txs, that's why we use 128<<20.
+const MEMTABLESIZE = 128 << 20
+
+// ReadTx implements the interface db.ReadTx
+type ReadTx struct {
+	tx *badger.Txn
+}
+
+// check that ReadTx implements the db.ReadTx interface
+var _ db.ReadTx = (*ReadTx)(nil)
+
+// WriteTx implements the interface db.WriteTx
+type WriteTx struct {
+	ReadTx
```

Comment:
hmm, why would a WriteTx contain a ReadTx? that means a WriteTx actually has two `*badger.Txn` pointers, but I assume it will only use one.

I think it would make sense to avoid this embedding, unless we require it for some reason.